### ISSUE TITLE
Install/Use CLI 

### ISSUE THREAD

**Issue by arquio** [xuorig/graphql-schema_comparator#8]
Hello @ xuorig, 

I try to use CLI but I am very new to RoR and I am very lost in what I can be doing wrong
 
My Log.
`/home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:34:in `block in print_changes': undefined method `breaking' for #<GraphQL::SchemaComparator::Changes::FieldRemoved:0x005638d348d4d0> (NoMethodError)
Did you mean?  breaking?
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:33:in `each'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:33:in `print_changes'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:23:in `compare'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:53:in `<top (required)>'
	from /home/dcalle/bin/graphql-schema:23:in `load'
	from /home/dcalle/bin/graphql-schema:23:in `<main>'`

Can you guide me?

Thanks a lot!!!

**Comment by rmosolgo** [comment]
Can you please share the command you ran which gave you this error?

**Comment by arquio** [comment]
[schemaB.graphql.txt](https://github.com/xuorig/graphql-schema_comparator/files/1458170/schemaB.graphql.txt)
[schemaA.graphql.txt](https://github.com/xuorig/graphql-schema_comparator/files/1458171/schemaA.graphql.txt)

`$ graphql-schema compare schemaA.graphql schemaB.graphql`
````
Output:
🤖  Checking for changes...
🎉  Done! Result:

Detected the following changes between schemas:

/home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:34:in `block in print_changes': undefined method `breaking' for #<GraphQL::SchemaComparator::Changes::FieldRemoved:0x0055dcfdf09ee0> (NoMethodError)
Did you mean?  breaking?
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:33:in `each'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:33:in `print_changes'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:23:in `compare'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
	from /home/dcalle/.gem/ruby/2.4.0/gems/thor-0.20.0/lib/thor/base.rb:466:in `start'
	from /home/dcalle/.gem/ruby/2.4.0/gems/graphql-schema_comparator-0.3.0/bin/graphql-schema:53:in `<top (required)>'
	from /usr/local/bin/graphql-schema:23:in `load'
	from /usr/local/bin/graphql-schema:23:in `<main>'
```

I'm test it in a clean heroku container with same result

**Comment by xuorig** [comment]
Thanks @arquio will fix this asap 👍

**Comment by xuorig** [comment]
Fixed in `0.3.1` https://github.com/xuorig/graphql-schema_comparator/releases/tag/v0.3.1

**Comment by arquio** [comment]
Perfect for this example
